doc/conf.py

```diff
@@ -1,11 +1,8 @@
 # -*- coding: utf-8 -*-
-import sys
-import os
-import glob
-import shutil
+# pylint: disable=invalid-name,redefined-builtin,redefined-builtin
+# pylint: disable=missing-module-docstring
```

Comment:
```suggestion
# -*- coding: utf-8 -*-
"""
Sphinx documentation configuration file.
"""
# pylint: disable=invalid-name
```

Also, if you can change this line:

https://github.com/GenericMappingTools/pygmt/blob/b2a1758555ce94465a29ab7318ca944c4074833c/doc/conf.py#L99

to use f-strings, and disable the `redefined-builtin` pylint warning:

```python
copyright = f"2017-{year}, The PyGMT Developers."  # pylint: disable=redefined-builtin
```

Reply:
Fixed in d94dbe7eb8beee1ccb14be3216dbc18c05f3800b.